transcend/types/data_silo.go

```diff
@@ -137,7 +137,7 @@ func MakeStandaloneUpdatePluginInput(d *schema.ResourceData) UpdatePluginInput {
 		Enabled:                  graphql.Boolean(d.Get("enabled").(bool)),
 		ScheduleFrequencyMinutes: graphql.String(strconv.Itoa(d.Get("schedule_frequency_minutes").(int) * 1000 * 60)),
 		ScheduleStartAt:          graphql.String(d.Get("schedule_start_at").(string)),
-		ScheduleNow:              graphql.Boolean(d.Get("schedule_now").(bool)),
+		ScheduleNow:              graphql.Boolean(false),
```

Comment:
Why has this been set to false? I guess cause we would never want to trigger a `scheduleNow` update through Terraform?

Reply:
That was my thought